skymp5-server/cpp/lib_espm/espm.cpp

```diff
@@ -566,15 +560,58 @@ const std::vector<espm::RecordHeader*>& espm::Browser::GetRecordsByType(
 }
 
 const std::vector<espm::RecordHeader*>& espm::Browser::GetRecordsAtPos(
-  uint32_t cellOrWorld, int16_t cellX, int16_t cellY)
+  uint32_t cellOrWorld, int16_t cellX, int16_t cellY) const
+{
+  const auto it =
+    pImpl->cellOrWorldChildren.find(RefrKey(cellOrWorld, cellX, cellY));
+  if (it == pImpl->cellOrWorldChildren.end()) {
+    const static std::vector<espm::RecordHeader*> kDefaultValue{};
```

Comment:
Global, static, and thread-local variables must be prefixed with `g_`: `static int g_foo`;